_docs/Threat-Model.md

```diff
@@ -1,70 +1,126 @@
+A storage eavesdropper can observe which prefixes are written by which IPs. Additionally it can guess by file size which files are downloaded by which IPs.
 
-A Drop Server can learn:
+###4.iv. Internet Eavesdropper
+Since only size and IP of requests to Qabel servers is observable an Internet eavesdropper can observe which IPs request which Qabel servers if it knows the IPs of Qabel servers (the request size is not fixed and thus not characteristic). By observing a great number of requests it might statistically guess which IPs communicate and share files among each other.
 
-1. A Drop Server learns how many distinct clients are using the server.
+###Worst Case Scenario
+Attacker *O* is contact of user *A*, can eavesdrop traffic at clients of user *A* and has full access to the Qabel servers *A* uses.
 
-### Qabel Storage Server
+This implies that *O* knows which storage server prefixes *A* uses. It also knows the number, size and modification time of the files, and the directory tree of *A's* prefixes. *O* can observe which IPs download files from *A's* prefixes and can guess which downloaders own the respective key. Additionally *O* can observe from which prefixes *A* downloads which files by matching the file size of the request and the stored files. The knowledge of *A's* drop ID is only a minor advantage to *O* since random users (can) write to *A's* drop ID. An attacker could statistically guess which IPs *A* communicates with by matching the IPs of downloaders from *A's* prefixes and senders of drop messages to *A's* drop ID.
 
-A Storage Server can learn:
+Visible Information:
 
-1. A Storage Server learns who owns a given Storage Volume.
-2. A Storage Server learns which IP addresses download a given Storage Blob.
-3. From the used URL a Storage Server learns who most likely uploaded a Storage Blob.
+**Storage**
+* *A* <-> *A's* prefixes
+* Number and size of *A's* files
+* Directory tree of *A's* folders
+* Actions (e.g., create file, share file, ...) *A* performs with respective
+    * Files and folders
+    * Time
+    * *IP(s) of destination drop server(s)*
+* Time and IP of downloads of *A's* files
+* Time, size, server IP and thereby possibly the prefix of *A's* downloads
 
-Problems:
+**Drop**
+* *A* -> Drop ID *A* listens
+* Time, size and sender IP of drop messages sent to *A's* drop ID
+* Time, size and drop server IP of drop messages *A* sends
 
-* A Storage Server should *not* learn that much about relations between users.
+####Worst Case Scenario under Usage of *Tor*
+Attacker *O* is contact of user *A*, can eavesdrop traffic at clients of user *A* and has full access to the Qabel servers *A* uses. This implies that *O* knows which storage server prefixes *A* uses. It also knows the number, size and modification time of the files, and the directory tree of *A's* prefixes.
 
-### A users Qabel Client
+Thus as far as all users additionally use *Tor* no information on relations between them is revealed:
```

Comment:
The *relation between users* is hidden since only the uploading user need to be authenticated and logged in. Thus the identity of a downloading user (which is not revealed) cannot be matched to the identity of the prefix owner.